ROADMAP.md

```diff
@@ -4,61 +4,88 @@
 - 0.5 - **Better customization.** 💅 We want to make our component customization intuitive, no matter if you are using global CSS or styled-components:
-  - Improve the documentation.
-  - Better support for styled-components.
-  - Work on the core solution.
-  - Provide [more examples like this](https://mui-treasury.com/components/button).
-- 0.3 - **Better documentation.** 📚 It's a wide topic. We don't know exactly what people want. We will focus our effort in the following areas:
-  - Beginner tutorials & Video lessons.
-  - Templates. They get people started really quickly, we need more of them!
+  - Better support for styled-components: [#6115](https://github.com/mui-org/material-ui/issues/6115).
+  - Allow the usage of the Box props in all the core components: [#15561](https://github.com/mui-org/material-ui/issues/15561).
+  - Allow the usage of dynamic theme variants and colors: [#15573](https://github.com/mui-org/material-ui/issues/15573) & [#13875](https://github.com/mui-org/material-ui/issues/13875).
+  - Allow the usage of the components without any styles: [#6218](https://github.com/mui-org/material-ui/issues/6218).
+  - Improve the suppor of custom breakpoints: [#11649](https://github.com/mui-org/material-ui/issues/11649)
+  - Explore the integration with theme-specification, by @jxnblk.
+- 0.3 - **Better documentation.** 📚 It's a wide topic. We will focus our effort in the following areas:
   - Page documentation rating 🥇🥈🥉. We will integrate a rating module in all our documentation pages. This way, we can collect high-quality data points and prioritize the pages that need the most to be improved.
-  - Localisation. Not everybody can read & understand English. The documentation pages are synced with [Crowdin](https://translate.material-ui.com/). They should always be up-to-date.
+  - Templates. They get people started really quickly, we need more of them!
+  - Beginner tutorials & Video lessons.
 - 0.3 - **TypeScript.** 📏 There are two dimensions to this problem:
-  - The documentation. We want to provide a TypeScript variant to most of our demos,
-    which should make it straightforward to use Material-UI with TypeScript. You can help us out in [#14897](https://github.com/mui-org/material-ui/issues/14897).
-  - The definitions. We are **continuously** improving them. We are aware that correctly typing withStyles() is cumbersome, so we encourage you to use [makeStyles()](/styles/basics/#hook-api) instead. It should be much better with this Hook API.
-  The codebase is written in JavaScript, we don't plan on migrating it to TypeScript in the near future. Maybe for v5, we will see.
-- 0.2 - **Performance.** 🚀 React abstraction has a cost. The more components you render, the slower your page will be. You will notice stark differences when rendering a large table or list.
+  - The documentation. We want to provide a TypeScript variant to most of our demos.
```

Comment:
Sorry mobile phone, no suggestions. ‘For most of our demos’ or ‘of most of our demos’?